sycl/source/ld-version-script.txt

```diff
@@ -3,16 +3,21 @@
   /* because vtable and typeinfo symbols make extern "C++" patterns more    */
   /* complicated than patterns against mangled names.                       */
   /*                                                                        */
-  /* With extern "C++" we have to match for "vtable for cl::sycl::foo", but */
-  /* not match for "vtable for std::__internal<cl::sycl::foo>".             */
+  /* With extern "C++" we have to match for "vtable for sycl::foo", but     */
+  /* not match for "vtable for std::__internal<sycl::foo>".                 */
 
   global:
-    /* Export everything from cl::sycl namespace */
+    /* Export everything from sycl namespace */
```

Comment:
The rules below this comment export everything from cl::sycl and __sycl_internal* namespaces!
I suggest add more details in the comment to clarify this difference.

Reply:
Removed this change from this patch, will make it a part of the second patch.